tests/queries/0_stateless/02133_classification.reference

```diff
@@ -0,0 +1,12 @@
+ru
+en
+fr
+ja
+zh
+{'ja':62,'fr':36,'un':0}
```

Comment:
What does this numbers mean?
Can we maybe convert them to probability?

Reply:
Also, are they separate predictions or correlated?

Reply:
It is kind of what share does certain language takes. I guess it is not 100 because there are always whitespace and punctuation signs which do not really belong to any particular language.